dts/bindings/sensor/avia,hx711.yaml

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) 2020 George Gkinis
+# Copyright (c) 2021 Jan Gnip
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  Avia HX711 sensor
+
+compatible: "avia,hx711"
+
+include: base.yaml
+
+properties:
+  sample-fetch-timeout-ms:
+    type: int
+    required: false
+    default: 700
```

Comment:
Not sure if `default` makes sense here since this value varies board to board.

Reply:
maybe increase it to 1000ms?
Seems sensible to add a "safe" default value so that the user does not have to attach a oscilloscope or logic analyser to find the value specific for his board.

If puts the effort of measuring his specific solution he can fine-tune.

If you insist of forcing the user to add this then ok.

Please verify.